test/spec/modules/conversantAnalyticsAdapter_spec.js

```diff
@@ -45,25 +48,33 @@ describe('Conversant analytics adapter tests', function() {
     };
     sandbox.stub(prebidGlobal, 'getGlobal').returns(getGlobalStub); // getGlobal does not seem to be available in testing so need to mock it
     clock = sandbox.useFakeTimers(DATESTAMP); // to use sinon fake timers they MUST be created before the interval/timeout is created in the code you are testing.
+
+    logInfoStub = sandbox.stub(utils, 'logInfo');/* .callsFake((arg, arg1, arg2) => { //debugging stuff
+      console.log(arg);
+      if (arg1) console.log(arg1);
+      if (arg2) console.log(arg2);
+    }); */
+
+    conversantAnalytics.enableAnalytics(VALID_ALWAYS_SAMPLE_CONFIG);
   });
 
   afterEach(function () {
     sandbox.restore();
     requests = []; // clean up any requests in our ajax request capture array.
+    conversantAnalytics.disableAnalytics();
   });
 
   describe('Initialization Tests', function() {
     it('should log error if site id is not passed', function() {
       sandbox.stub(utils, 'logError');
-
+      conversantAnalytics.disableAnalytics();
```

Comment:
Is this needed?

Reply:
Yes, was running the tests before and since I put in a global initialization to avoid putting it in for 90% of the test cases I have to reset here.